src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationsTest.java

```diff
@@ -32,8 +32,9 @@
 import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
 import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
 
-@Slf4j
 @RunWith(SpringIntegrationSerenityRunner.class)
+@WithTags({@WithTag("testType:Integration")})
+@Slf4j

```

Comment:
@WithTags({@WithTag("testType:Integration")}) you can add this tag in the superclass so that no need to add all the base classes

Reply:
already tried , its doesnt work

Reply:
following two tags needs to be together , then only tagging works, also if put it in super class then it requires at least 1 test case in that class.
@RunWith(SpringIntegrationSerenityRunner.class)
@WithTags({@WithTag("testType:Integration")})